server/src/main/java/io/pravega/schemaregistry/server/rest/ServiceConfig.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.schemaregistry.server.rest;
+
+import com.google.common.base.Strings;
+import io.pravega.auth.ServerConfig;
+import io.pravega.common.Exceptions;
+import lombok.Builder;
+import lombok.Getter;
+
+/**
+ * REST server config.
+ */
+@Getter
+@Builder
+public class ServiceConfig implements ServerConfig {
+    private final String host;
+    private final int port;
+    private final boolean tlsEnabled;
+    private final String tlsCertFile;
+    private final String tlsTrustStore;
+    private final String tlsKeyFilePath;
+    private final String tlsKeyPasswordFilePath;
```

Comment:
This is actually the keystore password. You might want to call it `tlsKeyStorePasswordFilePath`